src/NumericalAlgorithms/Interpolation/Callbacks/FindApparentHorizon.hpp

```diff
@@ -0,0 +1,200 @@
+// Distributed under the MIT License.
```

Comment:
Can this be done in a separate PR addressing #1095 ?

I think we need more discussion about where everything goes. `InterpolationTargetKerrHorizon` (not touched in this PR) also depends on `ApparentHorizons` (because it uses a Strahlkorper); should that be moved too?  Should the `ApparentHorizons` directory also be split up (we could have one directory for low-level infrastructure things like YlmSpherepack and Strahlkorper that will be used for wave extraction as well as AH finding, and one directory for things like FastFlow that explicitly find apparent horizons).

It would be much easier to review a PR that moves many files around but does nothing else.

Reply:
Ok. `Strahlkorper` could actuallly go into `DataStructures` or similar. As you say, it would be good to move reusable infrastructure out of `ApparentHorizons`.

I must say, I’m really looking forward to that PR 😄